Keen.NET.Test/DataSetTests_Integration.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Moq;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using Keen.Core;
+using Keen.Core.Dataset;
+using Keen.Core.Query;
+
+namespace Keen.Net.Test
+{
+    [TestFixture]
+    public class DataSetTests_Integration : TestBase
```

Comment:
I think we can swap `DataSetTests_Integration.cs` and `DatasetTests.cs` since, as far as our definition of "integration tests" goes, these are really unit tests and the other file has what we'd consider "unit tests." Also, minor nitpick: I want to rename to `DatasetTests_Integration.cs` with the same capitalization as the unit tests file.